src/main/scala/fi/vm/sade/hakurekisteri/hakija/HakijaActor.scala

```diff
@@ -426,8 +440,9 @@ class HakijaActor(hakupalvelu: Hakupalvelu, organisaatioActor: ActorRef, koodist
           puhelin = h.puhelin,
           sahkoposti = h.sahkoposti,
           kotikunta = h.kotikunta,
-          kansalaisuus = kansalaisuus,
-          kaksoiskansalaisuus = h.kaksoiskansalaisuus,
+          kansalaisuus = Some(kansalaisuus),
+          kaksoiskansalaisuus = if (kaksoiskansalaisuus.nonEmpty) Some(kaksoiskansalaisuus) else None,
+          kansalaisuudet = if (kaksoiskansalaisuus.nonEmpty) Some(List(kansalaisuus,kaksoiskansalaisuus)) else Some(List(kansalaisuus)),
```

Comment:
Pitäiskö tässä olla `kansalaisuudet = hakija.henkilo.kansalaisuudet...`?

Reply:
Joo, vaikuttaa että pitäis. Korjataan.